content/blog/2016/2016-05-25-gsoc-jenkins-web-ui-project.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "GSoC Project: New Job Creation and Configuration"
+- jenkins
+- core
+- jenkins2
+- gsoc
+author: samatdav
+---
+
+=== Abstract
+Although powerful, Jenkins new job creation and configuration process may be non obvious and time consuming. This can be improved by making UI more intuitive, concise, and functional. I plan to achieve this by creating a simpler new job creation, configuration process focused on essential elements, and embedding new functionality. 
+
+=== Deliverables
+==== New job creation
+* [Required] New job name validation
+** Sending GET request to doCheckJobName after each input character
+** Blocking item creation if it’s name is inappropriate
+* [Required] Popup window. http://i.imgur.com/6O3jcuA.gif[Here is a
+demonstrating animation]
+** Rendering a popup window with job creation +
+** Receiving JSON with job types
+** Sending a POST request with job name and type
+* [Optional] Simple new job configuration
+** Job creation popup contains step to preset some of configuration
+** Job creation popup sends a POST request to change confuguration on
+job creation
+
+==== Configuration page
+* [Required] Changing help information.
+http://i.imgur.com/zoSvB1G.gif[Here is a demonstrating animation.]
```

Comment:
I would also recommend to embed pictures into the blogpost. It would improve the visualization